Content.Shared/Hands/HandEvents.cs

```diff
@@ -231,4 +216,64 @@ public sealed class DidUnequipHandEvent : UnequippedHandEvent
     {
         public DidUnequipHandEvent(EntityUid user, EntityUid unequipped, Hand hand) : base(user, unequipped, hand) { }
     }
+
+    /// <summary>
+    /// A message that calls the use interaction on an item in hand, presumed for now the interaction will occur only on the active hand.
+    /// </summary>
+    [Serializable, NetSerializable]
+    public sealed class UseInHandMsg : EntityEventArgs
```

Comment:
these should still be `UseInHandEvent` by convention or something else renamed i think

Reply:
uh yeah I just moved them over from the other file but might as well rename them.